Documentation/Signoffs/Device_Power.md

```diff
@@ -0,0 +1,190 @@
+# Device Power Subsystem
+
+**Function:**
+
+![Function](../Images/Device_Power/Conceptual2.png)
+
+Figure 1: Device Power Subsystem
+
+The goal of this subsystem is to convert AC power from the wall outlet to DC power, and then distribute that power to the different device systems.  
+
+**Constraints:**
+
+| NO. | Constraint                                                          | Origin           |
+|-----|---------------------------------------------------------------------|------------------|
+| 1   | The power system shall be controlled by an emergency stop which will de-energize the device power system that will in turn de-energize the interceptor itself. |Conceptual Design |
```

Comment:
Even the controller? Is that a good idea?

Reply:
This was a Devcom constraint. They specifically requested the emergency stop deenergize the entire interceptor. The emergency stop is really only there if the system takes a wrong turn and needs to be turned for safety reasons.

Reply:
Sure. But that doesn't change the question.